packages/primeng/src/scroller/scroller.ts

```diff
@@ -715,38 +747,36 @@ export class Scroller extends BaseComponent implements OnInit, AfterContentInit,
 
     scrollTo(options: ScrollToOptions) {
```

Comment:
Since this is a public method, I didn't want to change its interface to expose some internal details of the component. However, I also wanted to keep the scrolling logic in one method/place, so I introduced a new private method to keep the control in one place and not expose internal details.